lib/event_tracker/tracker.rb

```diff
@@ -16,5 +22,10 @@ def context_with_(properties)
       @context.merge(properties)
               .merge({client_event_unix_timestamp: Time.now.to_i})
     end
+
+    # TODO: remove short circuit
```

Comment:
Can we avoid merging (solvable) TODO's?
Disclaimer: I am not sure what the current policy is on this.